dependencies.py

```diff
@@ -294,11 +307,9 @@ def _install_dependencies(dependencies, target_dir, use_pypi=False):
         "--target",
         str(target_dir),
     ]
-    if use_pypi:
-        index = base_command.index("--find-links")
-        base_command.pop(index)  # --find-links
-        base_command.pop(index)  # the dir
-        base_command.pop(index)  # --no-index
+
+    # append the dependency folder to the path
+    sys.path.append(target_dir)
```

Comment:
Hier is dus de grote vraag: append of prepend. Bij append kan er een verkeerde versie vóór staan.